src/components/ForumWidget/styles.module.scss

```diff
@@ -0,0 +1,35 @@
+.root {
+  display: flex;
+  font-size: 13px;
+  border: 1px solid #CCD0D5;
+  border-radius: 24px;
+  line-height: 16px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 8px 5px;
+  gap: 4px;
```

Comment:
Update layout so 'Date + Reading time' is centered when there is no linked Forum post.
```suggestion
  gap: 4px;
  margin-left: auto;
```

Reply:
@RyRy79261 We decided to make new changes, and this `margin-left: auto;` is no longer needed. Could you delete it? Thanks